no-computer.md

```diff
@@ -10,42 +10,59 @@ good way to meet their needs.
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
-
+My non-computer DSL would be hitchhiking/hailing taxis. This was needed before
+Uber and apps to signal to taxis and drivers that you needed a ride. It is a
+very simple language, with very few ways to signal, but it is very effective and
+widely recognized.
+I would create a few more indicators: 
+	- I am extremely desperate, will pay double.
+	- I have card only
+	- Default I just need a cab 
+This would make it more efficient.
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
-
+A language is certainly needed to communicate, even for this single purpose.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
-
+I used to take cabs often back home in Singapore, and drivers would often refuse
+to stop for no reason. It would be great if I could signal that I would tip or
+pay extra in rushed times.
 
 ### Domain
 _Describe the project's domain in five words._
-
+Better language for hailing cabs.
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
-
+A user sticks their hand out in a variety of ways. I imagine different angles
+of the arm. It is a good way because it needs to be clear and quick as people
+are driving when looking for hailing cabs.
 
 ### Operation (semantics)
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
+When a program runs, a cab may or may not stop if one passes by. Errors will be
+responded to with a guaranteed no-stop by a cab. 
```

Comment:
Lol. Hailing a cab doesn't seem to be very effective, but I guess there's no way to fix it